doc/programming_guide/install/obsolete.rst

```diff
@@ -1,11 +1,14 @@
-.. _currently supported by the PSF: https://devguide.python.org/#status-of-python-branches
+.. _currently supported by the PSF: https://devguide.python.org/versions/#supported-versions
 
 Installation for Obsolete Python Versions
 =========================================
 
 Arcade aims to support the same Python versions
-`currently supported by the PSF`_. You are strongly encouraged to upgrade to
-one of these if at all possible.
+`currently supported by the PSF`_.
+
+You are strongly encouraged to upgrade to one of the versions listed at the
+link above, with the exception of 3.11 or later. Some of arcade's dependencies
```

Comment:
Might be a good idea not not mention a specific version here to make the document relevant in the future?

Reply:
I haven't done it for arcade yet, but  I figured out a way to centralize minimum python versions in documentation. See https://github.com/pyglet/pyglet/pull/668 for an example. For arcade, I'd also add a `MAX_PYTHON_VERSION` constant. My intent was to make changes like this a separate PR.